app/models/vendor.rb

```diff
@@ -1,4 +1,5 @@
 class Vendor < ApplicationRecord
+  has_many :users
```

Comment:
This confused me at first, but now it makes sense.  Is this so you can identify a 'business_admin' by  the relationship to the vendor?  cool